src/plugin.js

```diff
@@ -514,24 +482,26 @@ class VR extends Plugin {
     this.getVideoEl_().style.display = 'none';
 
     if (window.navigator.getVRDisplays) {
+      this.log('VR is supported, getting vr displays');
       window.navigator.getVRDisplays().then((displays) => {
         if (displays.length > 0) {
-          this.log('WebVR supported, VRDisplays found.');
+          this.log('VR Displays found', displays);
           this.vrDisplay = displays[0];
-          this.log(this.vrDisplay);
+          this.log('Going to use VRControls on the first one', this.vrDisplay);
 
           // Native WebVR Head Mounted Displays (HMDs) like the HTC Vive
           // also need the cardboard button to enter fully immersive mode
           // so, we want to add the button if we're not polyfilled.
           if (!this.vrDisplay.isPolyfilled) {
             this.addCardboardButton_();
           }
-        // FIREFOX doesn't report the polyfill display as a
-        // VRDisplay so this would error even though the video would
-        // work
-        } else if (!videojs.browser.IS_FIREFOX) {
-          this.triggerError_({code: 'web-vr-no-devices-found', dismiss: false});
+          this.controls3d = new VRControls(this.camera);
+        } else {
+          this.log('no vr displays found going to use OrbitControls');
+          this.controls3d = new OrbitControls(this.camera, this.renderedCanvas);
```

Comment:
When no VRDisplay is found webvr-polyfill no longer inserts one for native vr implementations. So we have to do our own controls.